Source/WebCore/Modules/modern-media-controls/controls/abstract-slider.css

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2016,2022 Apple Inc. All Rights Reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY APPLE INC. ``AS IS'' AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL APPLE INC. OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+ * OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+.slider {
+    position: relative;
+}
+
+.slider > .slider-appearance,
+.slider > input {
+    position: absolute;
+    left: 0;
+    width: 100%;
+}
+
+.slider > .slider-appearance {
+    pointer-events: none;
+}
+
+.slider.allows-relative-scrubbing > .slider-appearance {
+    transition-property: top, height;
+    transition-duration: 0.5s;
+}
+
+.slider.allows-relative-scrubbing > .slider-appearance.changing {
+    top: 0.5px;
+    height: 15px;
```

Comment:
These make less sense now that the `top` and `height` are not defined in this file.  I'd either move the `.slider.default > .slider-appearance` from the other file back into this one (since it can always be overridden by a more specific selector from a subclass) or move this into the other file so that any `top` and `height` with exact values are always near eachother.

Reply:
I've moved these to `slider.css`.